controllers/module_reconciler.go

```diff
@@ -115,12 +112,6 @@ func (r *ModuleReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 		return res, fmt.Errorf("failed to get the requested %s KMMO CR: %w", req.NamespacedName, err)
 	}
 
-	if req.Namespace != r.operatorNamespace {
-		if err = r.caHelper.Sync(ctx, req.Namespace, mod); err != nil {
-			return ctrl.Result{}, fmt.Errorf("failed to synchronize CA ConfigMaps: %v", err)
-		}
-	}
```

Comment:
Why are we removing it?

Reply:
This helper was providing the CA certs to signing Jobs. We don't need it anymore because we use Builds, which mount the CA certs automatically.